policyengine_us/variables/gov/states/az/tax/income/exemptions/az_aged_exemption_eligible.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class az_aged_exemption_eligible(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Eligible for the Arizona aged exemption"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
+        separate = filing_status == filing_status.possible_values.SEPARATE
+
+        dependent_head = tax_unit("head_is_dependent_elsewhere", period)
+        head_eligible = ~dependent_head * separate
```

Comment:
The separate limitation does not match my read of the [law](https://www.azleg.gov/viewdocument/?docName=https://www.azleg.gov/ars/43/01023.htm):
>1. If the taxpayer has attained sixty-five years of age before the close of the taxable year filing a separate or joint return and the taxpayer is not claimed as a dependent by another taxpayer.

>2. For the taxpayer's spouse if the spouse has attained sixty-five years of age before the close of the taxable year, a joint return is filed and the spouse is not a dependent of another taxpayer.

Reply:
added